src/init.cpp

```diff
@@ -1048,6 +1048,9 @@ bool AppInitParameterInteraction()
     fAcceptDatacarrier = GetBoolArg("-datacarrier", DEFAULT_ACCEPT_DATACARRIER);
     nMaxDatacarrierBytes = GetArg("-datacarriersize", nMaxDatacarrierBytes);
 
+    nLocalServices = ServiceFlags(nLocalServices | NODE_SEGWIT2X);
```

Comment:
Question: Should this flag instead be set if and only if `NODE_WITNESS` is set (see code on line 1579)?

I don't know the answer as it depends on what the flag is used for.